MARS_QA/StepDefinitions/ProfileDescriptionStepDefinitions.cs

```diff
@@ -0,0 +1,64 @@
+using MARS_QA.Mars_Pages;
+using MARS_QA.Utilities;
+using NUnit.Framework;
+
+
+namespace MARS_QA.StepDefinitions
+{
+    [Binding]
+    public class ProfileDescriptionStepDefinitions : CommonDriver
+    {
+
+        [Given(@"user click on Pen button for Description")]
+        public void GivenUserClickOnPenButtonForDescription()
+        {
+            login();
+        }
+
+        [When(@"user enter Description and click Save Button '([^']*)'")]
+        public void WhenUserEnterDescriptionAndClickSaveButton(string description)
+        {
+            ProfileDescription.AddProfileDescription(description);
+        }
+
+        [Then(@"Description is saved successfully")]
+        public void ThenDescriptionIsSavedSuccessfully()
+        {
```

Comment:
you should be asserting in thens please add asserts!

Reply:
you should never have empty steps it is bad practice! A step should always do something!